components/FavouriteIcon.js

```diff
@@ -1,17 +1,14 @@
 import { useState } from "react";
 
-export default function FavoriteIcon() {
-  const [isFavorite, setIsFavorite] = useState(false);
-  function handleToggle() {
-    setIsFavorite(!isFavorite);
-  }
-
+export default function FavoriteIcon({ isFavorite }) {
+  console.log(isFavorite);
```

Comment:
`console.logs`should be removed before merging into main

Reply:
thx i fixed it now 🙏